### DISCUSSION TITLE
Will HotReload be available later?

### DISCUSSION THREAD

**Discussion by lukewire129** [CommunityToolkit/Maui.Markup#261] [Q&A] [answered]
![image](https://github.com/CommunityToolkit/Maui.Markup/assets/54387261/24816916-6393-442b-8100-ea19c53629bc)

seeing that it's supported 
I received the update and made a sample and tested it.
However, it didn't work.

UseMauiCommunityToolkitMarkup

We have seen that RegisterReloadApplicationEventHandler() is missing.

Is it on hold for a while?

[Git Sample]
![image](https://github.com/CommunityToolkit/Maui.Markup/assets/54387261/d7f667e0-afdc-44ad-a497-5c3a9c5f400e)

[Nuget -Reference]
![image](https://github.com/CommunityToolkit/Maui.Markup/assets/54387261/ceb33a08-e1a7-4b9d-8e39-a15707a5df6a)

**Comment by TheCodeTraveler** [comment, marked as answer]
Good catch! I've opened #263 to remedy this bug and will release it in v3.3.1.

**Reply by lukewire129** [reply]
Thank you! 
I'll try again after I finish my personal work!
